src/mds/objects.py

```diff
@@ -60,7 +60,6 @@ class CreateObjForIdInput(BaseModel):
     Create object.
 
     file_name (str): Name for the file being uploaded
-    authz (dict): authorization block with requirements for what's being uploaded
```

Comment:
why'd this get removed?

Reply:
I noticed that the `CreateObjForIdInput ` model doesn't have an `authz` field. So even if a user were to `POST /objects/{GUID}` with `authz` provided in the body, `authz` of the created record and metadata object would be copied from the previous record's `authz` field, not set based on the `POST`-provided `authz`.